rllib/utils/torch_ops.py

```diff
@@ -10,11 +10,8 @@
 def explained_variance(y, pred):
     y_var = torch.var(y, dim=[0])
     diff_var = torch.var(y - pred, dim=[0])
-    min_ = torch.Tensor([-1.0])
-    return torch.max(
-        min_.to(device=torch.device("cuda"))
-        if torch.cuda.is_available() else min_,
-        1 - (diff_var / y_var))
+    min_ = torch.tensor([-1.0]).to(pred.device)
```

Comment:
This was a bug (should only use GPU if num_gpus > 0, not whenever one is available).